puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -267,15 +269,23 @@
     security_group_api     => $security_group_api,
   }
 
+  if !$enable_vif_type_n1kv {
```

Comment:
Minor comment, this would be easier to read if you dropped the negation and instead made it:
```
if $enable_vif_type_n1kv {
  $_interface_driver = "neutron.agent.linux.interface.N1KVInterfaceDriver"
} else {
  $_interface_driver = "neutron.agent.linux.interface.OVSInterfaceDriver"
}
```